pkg/kubelet/cm/dra/manager.go

```diff
@@ -282,8 +284,8 @@ func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 				resourceHandle.Data)
 			if err != nil {
 				return fmt.Errorf(
-					"NodeUnprepareResource failed, pod: %s, claim UID: %s, claim name: %s, CDI devices: %s, err: %+v",
-					pod.Name, claimInfo.ClaimUID, claimInfo.ClaimName, claimInfo.CDIDevices, err)
+					"NodeUnprepareResource failed, pod: %s, claim UID: %s, claim name: %s, resource handle: %s, err: %+v",
+					pod.Name, claimInfo.ClaimUID, claimInfo.ClaimName, resourceHandle.Data, err)
```

Comment:
`claimInfo` properties are referenced here and above. Why don't we lock it?

Reply:
The claimInfo is created in [1] and during the lifetime of the PrepareResources/GetResources/UnprepareResources we write only for following fields: PodUIDs (deletePodReference) , CDIDevices (addCDIDevices)  and annotations( addCDIDevices). 
So In my opinion we should only guard them. 

[1] - https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/dra/manager.go#L112-L119